Migration.Executor.WebJob/Properties/launchSettings.json

```diff
@@ -0,0 +1,16 @@
+{
+  "profiles": {
+    "Migration.Executor.WebJob": {
+      "commandName": "Project",
+      "environmentVariables": {
+        "cosmosdbaccount": "fabianm-mig-dev-cdb",
+        "cosmosdbleasescollection": "Leases",
+        "keyvaulturi": "https://fabianmmikv3pyyeib2q7sis.vault.azure.net/",
```

Comment:
oops, might want to undo this commit history?

Reply:
It is ok- uses AAD auth and only will work for my account